### PR TITLE
add delete edit features and diagrams

### PR DESCRIPTION
**Pull request by LimAiLin**

### CODE REVIEW

File: docs/diagrams/AddStudentSequenceDiagram.puml [hunk 1]
```diff
@@ -25,7 +25,7 @@ activate AddStudentCommandParser
 AddStudentCommandParser --> ModQuikParser
 deactivate AddStudentCommandParser
 
-ModQuikParser -> AddStudentCommandParser : parse("")
+ModQuikParser -> AddStudentCommandParser : parse("n/Name i/A0000000A \n ph/88888888 e/example@example.com \n tele/myHandle m/CS2103T tut/w17")
```
Comment (lilythchu) [inline docs/diagrams/AddStudentSequenceDiagram.puml:28]: have to deactive LogicManager

File: docs/diagrams/EditStudentSequenceDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+Model --> EditStudentCommand
+deactivate Model
+
+EditStudentCommand --> LogicManager : result
+deactivate EditStudentCommand
+
+EditStudentCommand -[hidden]-> LogicManager
+destroy EditStudentCommand
+
+[<--LogicManager
```
Comment (lilythchu) [inline docs/diagrams/EditStudentSequenceDiagram.puml:79]: should deactive LogicManager

Comment (LimAiLin) [reply docs/diagrams/EditStudentSequenceDiagram.puml:79]: done!

File: docs/diagrams/AddStudentSequenceDiagram.puml (hunk 1 above)
Comment (LimAiLin) [reply docs/diagrams/AddStudentSequenceDiagram.puml:28]: done!

**Review by lilythchu — APPROVED** [review]

**Pull request merged**